cli/sign.go

```diff
@@ -61,6 +61,20 @@ func signExisting(c *cli.Context) error {
 	}
 	defer f.Close()
 
+	artFileStat, err := os.Stat(artFile)
+	if err != nil {
+		return cli.NewExitError("Could not get artifact file stat", 1)
+	}
+	err = os.Chown(tFile.Name(), int(artFileStat.Sys().(*syscall.Stat_t).Uid),
```

Comment:
Question: Why don't you use the existing getters for the `os.Stat` structure? I see it has functions like this: https://pkg.go.dev/os#Getgid

Reply:
According to my understanding, os.Getgid only returns the gid of the caller, and not of a specific file.

Reply:
Darn... Ur right of course. I thought it was a method of the returned stat struct. But of course it's not. Good calling me out :) 

I seem to recall you were having some issues with the windows cross compilation though, and that is most likely due to this direct access I would guess.

Have you done any more research on it?

Reply:
Then i would be very impressed :dark_sunglasses: